modules/craftBidAdapter.js

```diff
@@ -132,12 +130,17 @@ function newBid(serverBid, rtbBid, bidderRequest) {
 
 function bidToTag(bid) {
   const tag = {};
-  for (var k in bid.params) {
+  for (const k in bid.params) {
     tag[k] = bid.params[k];
   }
   try {
     if (storage.hasLocalStorage()) {
-      tag.uid = JSON.parse(storage.getDataFromLocalStorage(`${bid.params.sitekey}_uid`));
+      const field = `${bid.params.sitekey}_uid`;
+      tag.uid = JSON.parse(storage.getDataFromLocalStorage(field));
+      if (!tag.uid) {
+        tag.uid = {value: {id: generateUUID()}};
+        storage.setDataInLocalStorage(field, JSON.stringify(tag.uid));
```

Comment:
declare ion your docs pr exactly which key you are setting and what its purpose is, eg in a format similar to https://business.safety.google/adscookies/